build/images/ovs/build.sh

```diff
@@ -19,23 +19,21 @@
 
 set -eo pipefail
 
-function echoerr {
-    >&2 echo "$@"
-}
+THIS_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+
+source $THIS_DIR/../build-utils.sh
 
 _usage="Usage: $0 [--pull] [--push] [--platform <PLATFORM>] [--distro [ubuntu|ubi]]
 Build the antrea openvswitch image.
         --pull                  Always attempt to pull a newer version of the base images
         --push                  Push the built image to the registry
         --platform <PLATFORM>   Target platform for the image if server is multi-platform capable
-        --distro <distro>       Target Linux distribution"
-
-function print_usage {
-    echoerr "$_usage"
-}
+        --distro <distro>       Target Linux distribution
+        --no-cache              Do not use the local build cache nor the cached image from the registry"
```

Comment:
For the new `--no-cache` mode, should we add some logic in CI to enable it when the related Dockerfile is changed to automatically verify base image changes without manually adding this parameter?

Reply:
There should be no need to provide the flag in this case AFAIK. A change to the Dockerfile should invalidate the cache as appropriate.